plugins/qtosgrave/osgviewerwidget.cpp

```diff
@@ -279,12 +279,28 @@ class OpenRAVETrackball : public osgGA::TrackballManipulator
     // make zooming faster
     bool performMovementRightMouseButton( const double eventTimeDelta, const double dx, const double dy )
     {
-        if(_posgviewerwidget->IsInOrthoMode()) {
-            _posgviewerwidget->Zoom( dy < 0 ? 0.9 : 1.1 );
-            return true;
+        bool result = osgGA::TrackballManipulator::performMovementRightMouseButton(eventTimeDelta, dx, dy*2);
+        // adjust camera perspective if zoom changed while in ortho view
+        if (_posgviewerwidget->IsInOrthoMode()) {
```

Comment:
Am debating using a variant of what is already in production, but I'll have to recompile the code first in order to be able to test.